src/mame/audio/taito_zm.h

```diff
@@ -14,12 +14,15 @@
 #include "cpu/tms57002/tms57002.h"
 #include "sound/zsg2.h"
 
+#define USE_DSP               // Uncomment when DSP emulation is working
```

Comment:
This name is far too generic for something in a header that needs to be included in downstream driver sources.  Could someone please either remove it completely if the DSP is in fact working, or at least prefix it with something to make it less likely to class with something equally generically named in another header?